airflow/models/taskinstance.py

```diff
@@ -1512,9 +1513,24 @@ def _run_raw_task(
         if not test_mode:
             session.add(Log(self.state, self))
             session.merge(self)
-
+            self._create_dataset_dag_run_queue_records(session=session)
             session.commit()
 
+    def _create_dataset_dag_run_queue_records(self, *, session=NEW_SESSION):
```

Comment:
```suggestion
    def _create_dataset_dag_run_queue_records(self, *, session):
```

(since it is a required value)